kotlin-react-function-plugin/src/main/kotlin/com/bnorm/react/ReactFunctionCallTransformer.kt

```diff
@@ -237,14 +237,14 @@ class ReactFunctionCallTransformer(
   // TODO better name?
   private fun IrBuilderWithScope.irCall_functionalComponent(propsType: IrType, name: String, body: IrBlockBodyBuilder.(IrSimpleFunction) -> Unit): IrCall {
     // TODO type=@[ExtensionFunctionType]?
-    val lambdaType = context.irBuiltIns.function(2)
-      .createType(false, listOf(
-        classes.react.RBuilder as IrTypeArgument,
-        propsType as IrTypeArgument,
-        context.irBuiltIns.unitType as IrTypeArgument
-      ))
-
-    return irCall(functions.react.fc, classes.react.FC(propsType)).apply {
+    val lambdaType = context.irBuiltIns.functionN(2)
```

Comment:
I'm guessing this change is because of some change in the compiler?

Reply:
When I updated to Kotlin 1.6, there were unresolved functions; I used IntelliJ auto complete to get the right methods to call, since there's no documentation